src/components/ArticleList.vue

```diff
@@ -91,12 +90,11 @@
         }
         return [...Array(Math.ceil(this.articlesCount / this.itemsPerPage)).keys()].map(e => e + 1)
       },
-      ...mapGetters({
-        articlesCount: [GET_ARTICLE_COUNT],
-        isLoading: [GET_ARTICLES_IS_LOADING],
-        isAuth: [IS_AUTHENTICATED],
-        articles: [GET_ARTICLES]
-      })
+      ...mapGetters([
+        `articlesCount`,
```

Comment:
any particular reason to use string templates in `mapGetters`?

Reply:
@atilacamurca @disjfa I had the same question when looking over the PR. The [Standard JavaScript Code Style](https://github.com/standard/standard) requires usage of single quotes for strings and template strings only when we need to intertwine with variables, but this will most likely never happen in this situation, unless we need to dynamically compute getter names.

Reply:
Just the "it worked" answer. Will check the single quote type, didn't check really.